Gemfile.lock

```diff
@@ -14,6 +14,7 @@ GEM
     ethon (0.11.0)
       ffi (>= 1.3.0)
     ffi (1.9.25)
+    ffi (1.9.25-x64-mingw32)
```

Comment:
I am not sure that we want to change our gemfile with this PR. Was there a reason for the update?